CMakeLists.txt

```diff
@@ -1,155 +1,143 @@
 ### CMakeList for MC Hydra
 
-cmake_minimum_required(VERSION 3.0)
-cmake_policy(SET CMP0054 OLD)
-
-# project name
-project(Hydra)
-
-# The version number.
-set (Hydra_VERSION_MAJOR 1)
-set (Hydra_VERSION_MINOR 0)
-set (Hydra_VERSION_PATCH 0)
```

Comment:
Shouldn't this be 2.0.0? These names are auto-generated if you set the VERSION keyword in project.

Reply:
Ideally, should pick-up the correct version from Hydra.h as doxygen does.  Can you implement this?

Reply:
Sure, will be able to finish this later today.